modules/navigation/3d/nav_mesh_queries_3d.h

```diff
@@ -71,6 +71,10 @@ class NavMeshQueries3D {
 		PathPostProcessing path_postprocessing = PathPostProcessing::PATH_POSTPROCESSING_CORRIDORFUNNEL;
 		bool simplify_path = false;
 		real_t simplify_epsilon = 0.0;
+		bool exclude_regions = false;
+		bool include_regions = false;
+		LocalVector<RID> excluded_regions;
```

Comment:
Is there really a way to have both of those active at the same time ? Seems like we either exclude region or include region, but doing both at the same time seems ... odd ? If this is the case, all could be siplified, as we only need one local vector and all those previous if could become switch cases.

Reply:
This will just happen with users that both lists are used and some RID slips in both lists. That is how it always goes with users when both include and exclude options are available so better account for it.